arrocke/gloss-translation
# feat: approve all glosses

## What has changed

- New `Approve All` button in toolbar and new shortcut `Alt+A` for translators
- New api path: ~~`PATCH /languages/[code]/verses/[verseId]/glosses`~~
  - Updated: `PATCH /languages/[code]/glosses/bulk`

## Connected Issues
closes #357 

## QA Steps
1. Go to a verse. Make sure you are logged out. 
2. Confirm that no  approve all button appears and Alt+A does nothing
3. Log in as admin.
4. Confirm that the button appears.
5. Enter some glosses. 
6. Approve all. Confirm that they are approved by the action
7. Unapprove a gloss. Confirm the button is enabled now.
8. Repeat steps 6 & 7 using Alt+A instead of the button.
## Post-Deployment
